[PnP] Overview (#2189)
OfficeDev/office-js-docs-pr: docs/overview/patterns-and-practices-overview.md

---

> title: Overview of Patterns and Practices for Office Add-ins

Reviewer: ```suggestion
title: Overview of Patterns and Practices (PnP) for Office Add-ins
```
I think this will improve SEO, search terms are high for "PnP"

> description: 'Overview of Patterns and Practices content for Office Add-in developers.'

Reviewer: ```suggestion
description: 'Overview of Patterns and Practices (PnP) content for Office Add-in developers.'
```

ms.date: 09/11/2020
localization_priority: Priority
---

> # Overview of Patterns and Practices for Office Add-ins

Reviewer: ```suggestion
# Overview of Patterns and Practices (PnP) for Office Add-ins
```

> Office Add-ins PnP is a community driven effort that helps developers extend, build, and deploy solutions for the Office platform. The source is maintained on the [PnP-OfficeAddins GitHub repo](https://github.com/OfficeDev/PnP-OfficeAddins) where anyone can participate. You can provide contributions to the samples, and documentation. Office Add-ins PnP is owned and coordinated by Office engineering teams, but the work is done by the community for the community. Office Add-ins PnP is a part of the broader [Microsoft 365 & SharePoint Ecosystem](https://developer.microsoft.com/office/blogs/microsoft-365-sharepoint-ecosystem-pnp-august-2020-update/).

Reviewer 2: ```suggestion
Office Add-ins PnP is a community-driven effort that helps developers extend, build, and deploy solutions for the Office platform. The source is maintained on the [PnP-OfficeAddins GitHub repo](https://github.com/OfficeDev/PnP-OfficeAddins) where anyone can participate. You can contribute to the samples and documentation. Office Add-ins PnP is owned and coordinated by Office engineering teams, but the work is done by the community for the community. Office Add-ins PnP is part of the broader [Microsoft 365 & SharePoint Ecosystem](https://developer.microsoft.com/office/blogs/microsoft-365-sharepoint-ecosystem-pnp-august-2020-update/).
```

> See Also

Reviewer 2: ```suggestion
See also
```
